### PR TITLE
always init bias with zeros and free tokens from weight decay

### PR DESCRIPTION
**Pull request by BingzhaoZhu**
*Issue #, if available:*

*Description of changes:*
1. According to the code from [FT_Transformer authors](https://github.com/puhsu/tabular-dl-pretrain-objectives/blob/master/lib/deep.py#:~:text=def%20default_zero_weight_decay_condition(module_name%2C%20module%2C%20parameter_name%2C%20parameter)%3A), tokenizers should be free from weight decay.  Worked slightly better on classification tasks (57% win rate).

2. Additive attention always init bias with zeros.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by sxjscience** [comment]
LGTM overall. Will it improve the performance of the model.

**Review by sxjscience — APPROVED** [review]
LGTM

**Review by gradientsky — APPROVED** [review]
